synapseclient/client.py

```diff
@@ -506,6 +506,8 @@ def _loggedIn(self):
                 if user['displayName'] == 'Anonymous':
                     return False
                 return user['displayName']
+            elif 'userName' in user:
```

Comment:
Do you use this function in this PR?  If not, it may be faster to pull the bug fix in another PR.

Reply:
This was intended to unblock Anthony (see [here](https://github.com/Sage-Bionetworks/synapsePythonClient/pull/894#issuecomment-1034176703)). I don't know if this issue with `_loggedIn()` come up elsewhere.